src/common/helper/env.helper.ts

```diff
@@ -0,0 +1,15 @@
+import { existsSync } from 'fs';
+import { resolve } from 'path';
+
+export function getEnvPath(val: string): string {
+  const env: string | undefined = process.env.NODE_ENV;
+  const fallback: string = resolve(`${val}/.env`);
```

Comment:
Let's not have a fallback. Exit the process with non-zero status code with appropriate error log in stderr.